### PR TITLE
First draft of .BANK article

### PR DESCRIPTION
**Pull request by aeden**
TLD-specific article for .BANK.

### CODE REVIEW

**Comment by aeden** [comment]
Requesting a review from @itsalyse for editorialization and from @ggalmazor for technical review.

File: content/articles/domains-bank.markdown [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+3. All .BANK web sites must be secured with HTTPS
+4. All .BANK web servers must use TLS 1.2 or higher
+5. All .BANK domain names must include SPF and DKIM records if they are used to send email
+6. All vendors that are providing web and mail hosting services must follow the same DNSSEC and TLS requirements
+
+### How DNSimple Helps You
+
+DNSimple implements these security requirements as follows:
+
+1. To use .BANK domaions with DNSimple the managing account must be on the DNSimple Business plan, which includes vanity name servers. The vanity name servers feature enables the use of your own domain as the NS records for your zone. For example, domain.bank will have the ability to use ns1.domain.bank, ns2.domain.bank, ns3.domain.bank, and ns4.domain.bank, mapped to the DNSimple name servers.
```
Comment (ggalmazor) [inline content/articles/domains-bank.markdown:34]:
```suggestion
1. To use .BANK domains with DNSimple the managing account must be on the DNSimple Business plan, which includes vanity name servers. The vanity name servers feature enables the use of your own domain as the NS records for your zone. For example, domain.bank will have the ability to use ns1.domain.bank, ns2.domain.bank, ns3.domain.bank, and ns4.domain.bank, mapped to the DNSimple name servers.
```

**Review by ggalmazor — APPROVED** [review]
LGTM. I checked the various references on ftld.com and register.bank to verify that the provided information matches official information.

**Review by itsalyse — APPROVED** [review]

File: content/articles/domains-bank.markdown [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+3. You can acquire Let's Encrypt certificates through DNSimple at no additional charge. Let's Encrypt certificates are invalidated every 90 days, so you will need a process to ensure the latest certificate is installed in any of your web servers when new certificates are generated. If you use our HTTPS redirection service, Let's Encrypt certificates can be automatically generated and will be used for the redirection of your domain name.
+4. TLS settings must be configured in your web server. If you use our HTTPS redirection service, HTTP requests will be automatically encrypted using TLS 1.2 or higher.
+5. SPF and DKIM records are implemented as TXT records in your zone, and DNSimple fully supports them. 
+6. DNSimple's URL redirection service is compliant with the requirements set forth by the .BANK registry for handling HTTPS requests over TLS.
+
+### Security reference
+
+For further reading about .BANK security requirements, the .BANK registry provides a comprehensive [Technical Guide to Security Requirements](https://go.ftld.com/full-documentation).
```
Comment (ramamehrjoo) [inline content/articles/domains-bank.markdown:43]: R

**Pull request merged**